osabie.py

```diff
@@ -2840,13 +2840,25 @@ def run_program(commands,
                 counter_variable.pop()
                 counter_variable.append(a)
 
+            elif current_command == ".\u00bc":
+                a = pop_stack(1)
+                stack.append(single_vectorized_evaluation(
+                    a, lambda a: math.tan(a)
```

Comment:
There needs to be some kind of evaluation here in order to also convert strings to ints (with `ast_int_eval`) or else it will not work for values like `"0.5"`.